File: src/Model/Seer.java
package Model;

import Technical.PowerEnum;

public class Seer extends GameCharacter {
    public Seer() {
        super("Seer",
                "The Seer can see the card of another player one time per turn.",
                true,
                PowerEnum.Constant,
                "../../img/Thiercelieux.png");
    }
}

File: app/src/main/java/Model/Seer.kt
package Model

class Seer(order: Int)
    : Character("The Seer can see the card of another player one time per turn."
    , true
    , PowerState.PERMANENT
    , false
    , order
    , 1) {

    override fun action(): String {
        return "The $className choose a card to see."
    }

    override fun clone() = Seer(this.order)
}